src/styles/_mixins.scss

```diff
@@ -70,3 +78,15 @@ $shadow-color: rgba(0, 0, 0, 0.2);
 
   z-index: 1;
 }
+
+@mixin tablet {
+  @media (min-width: #{$tablet-width}) {
```

Comment:
I don't think we can use media queries, our components can be much smaller than the screen. We need to use useElementDimensions to take the width and add classes conditionally.

Reply:
I reversed these changes.